RedHat-UX/red-hat-design-system
# docs(uxdot-copy-button): add accessible labels to copy buttons

## What I did

1. Added accessible names to the `<uxdot-copy-buttons>` used on the Tokens pages and Code pages for elements.
2. Fixed a quadruple dash (`----`) appearing in front of variable names on the element Code Design Tokens table.
    * See this [in production](https://ux.redhat.com/elements/call-to-action/code/#usage) by scrolling down to the Design Tokens table in the accordion and checking out the Copy icon/tooltip.

## Testing Instructions

1. Visit one of the [tokens pages](https://deploy-preview-1998--red-hat-design-system.netlify.app/tokens/color/) in the DP.
4. Inspect the copy icons and link icons.
5. Enter "Accessibility Tree View" in DevTools.
6. Verify that each copy and link icon have an accessible name.
7. Now visit any element's [Code page](https://deploy-preview-1998--red-hat-design-system.netlify.app/elements/call-to-action/code/#usage) (let's do CTA).
8. Scroll down to the Design Tokens table where there are Copy and Link icon's.
9. Verify those icons/buttons have accessible names.
10. Yes? ✅ 

## Notes to Reviewers

Enjoy newly accessible click to copy buttons.
